static/js/components/ObservingRunPage.jsx

```diff
@@ -115,14 +156,23 @@ const ObservingRunList = ({ observingRuns }) => {
 
 const ObservingRunPage = () => {
   const { observingRunList } = useSelector((state) => state.observingRuns);
```

Comment:
Are you sure you want to call the variable holding the observing runs `observingRunList` the same as the name of one of the components? maybe just use `observingRuns`?

Reply:
It is already used in other places so probably too late to switch now.

Reply:
Seems very confusing to me, but fine.